acta-universitatis-agriculturae-et-silviculturae-mendelianae-brunensis.csl

```diff
@@ -0,0 +1,595 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
```

Comment:
@matejovo, since https://acta.mendelu.cz/for-authors.html says "Contributions are published only in English.", I assume this style can be fixed to British or US English? See https://github.com/citation-style-language/styles/wiki/Style-Requirements#3---style-locale